docs/configuration.md

```diff
@@ -167,3 +167,36 @@ Any flag that has a value that is a comma separated list on the command line bec
     [profile]
     cpu-time = "30s"
     ```
+##### Metric Service
+* Description: which stats service to use (StatsD or ExpVar)
```

Comment:
make this consistent with the others (i.e start with capital letter and end with a period)